hooks/useImageManipulation.ts

```
@@ -0,0 +1,146 @@
+import { useCallback } from 'react';
+import { toast } from 'sonner';
+import clientUploadToArweave from '@/lib/arweave/clientUploadToArweave';
+
+interface ImageTransform {
```

ESLint

[error] 'ImageTransform' is defined but never used.

(@typescript-eslint/no-unused-vars)